test-curves/src/mnt6_753/mod.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(feature = "mnt6_753")]
```

Comment:
This feature flag only flags the existence of the `fr` module; not the other modules or re-exports; is this the intended behaviour?

Reply:
good catch! thx